---
# MIOP terms
methodology_category: omics analysis
project: "NOAA Ocean Exploration seawater eDNA metabarcoding"
purpose: PCR [OBI:0000415]
analyses: PCR [OBI:0000415]
geographic_location: Atlantic Ocean [GAZ:00000344], Gulf of Mexico [GAZ:00002853], Pacific Ocean [GAZ:00000360]
broad_scale_environmental_context: marine biome [ENVO:00000447], marine photic zone [ENVO:00000209]
local_environmental_context: marine biome [ENVO:00000447], marine photic zone [ENVO:00000209]
environmental_medium: sea water [ENVO:00002149]
target: 12S mitochondrial ribosomal RNA  [NCIT:C128263]
creator: Steven Auscavitch, Allen Collins
materials_required: vortexer [OBI:0400118], PCR instrument [OBI:0000989], agarose gel electrophoresis system [OBI:0001134]
skills_required: sterile technique, pipetting skills, standard molecular technique
time_required: 240
personnel_required: 1
language: en
issued: 2025-12-19
audience: scientists
publisher: Smithsonian NMNH
hasVersion: 1
license: CC0 1.0 Universal
maturity level: mature

# FAIRe terms
pcr_0_1: 1
thermocycler: Eppendorf Mastercycler Nexus Thermal Cycler
amplificationReactionVolume: 25
assay_name: Fish-12S-MiFish-Miya
assay_validation: not provided
targetTaxonomicAssay: Actinopterygii
targetTaxonomicScope: "Also captures non-target: Vertebrates, bacteria"
target_gene: 12S rRNA (SSU mitochondria)
target_subfragment: V5-V6
ampliconSize: 163-185
pcr_primer_forward: GTCGGTAAAACTCGTGCCAGC
pcr_primer_reverse: CATAGTGGGGTATCTAATCCCAGTTTG
pcr_primer_name_forward: MiFish-U-F
pcr_primer_name_reverse: MiFish-U-R
pcr_primer_reference_forward: http://dx.doi.org/10.1098/rsos.150088
pcr_primer_reference_reverse: http://dx.doi.org/10.1098/rsos.150088
pcr_primer_vol_forward: not provided
pcr_primer_vol_reverse: not provided
pcr_primer_conc_forward: not provided
pcr_primer_conc_reverse: not provided
probeReporter: not applicable
probeQuencher: not applicable
probe_seq: not applicable
probe_ref: not applicable
probe_conc: not applicable
commercial_mm: not provided
custom_mm: not provided
pcr_dna_vol: not provided
pcr_rep: not provided
nucl_acid_amp: not provided
pcr_cond: not provided
annealingTemp: not provided
pcr_cycles: not provided
pcr_analysis_software: not provided
pcr_method_additional: not provided
barcoding_pcr_appr: two-step PCR
pcr2_thermocycler: not provided
pcr2_amplificationReactionVolume: not provided
pcr2_commercial_mm: not provided
pcr2_custom_mm: not applicable
pcr2_dna_vol: not provided
pcr2_cond: not provided
pcr2_annealingTemp: not provided
pcr2_cycles: not provided
pcr2_analysis_software: not applicable
pcr2_method_additional: not applicable
---

# NMNH PCR Protocol 12S rRNA MiFish (Miya)

***WARNING: DETAILS BELOW HAVE NOT BEEN UPDATED. PLEASE REFER TO YAML FRONT MATTER ABOVE.***

## PROTOCOL INFORMATION

### Minimum Information about an Omics Protocol (MIOP)

- MIOP terms are listed in the YAML frontmatter of this page.
- See <https://github.com/BeBOP-OBON/miop/blob/main/model/schema/terms.yaml> for list and definitions.

### Making eDNA FAIR (FAIRe)

- FAIRe terms are listed in the YAML frontmatter of this page.
- See <https://fair-edna.github.io/download.html> for the FAIRe checklist and more information.
- See <https://fair-edna.github.io/guidelines.html#missing-values> for guidelines on missing values that can be used for missing FAIRe or MIOP terms.

### Authors

| PREPARED BY | AFFILIATION | ORCID | DATE |
| ------------- | ------------- | ------------- | ------------- |
| Allen Collins | Smithsonian NMNH | <https://orcid.org/0000-0002-3664-9691> | missing |
| Steven Auscavitch | Smithsonian NMNH |  <https://orcid.org/0000-0001-5777-4814> | missing |
| Luke Thompson | NOAA/AOML, MSU/NGI | <https://orcid.org/0000-0002-3911-1280> | 2025-12-19 |
| Sammy Harding | NOAA/AOML, MSU/NGI | <https://orcid.org/0009-0008-8885-6140> | 2025-12-19 |

## RELATED PROTOCOLS

| PROTOCOL NAME | LINK  | VERSION | RELEASE DATE |
| ------------- | ------------- | ------------- | ------------- |
| NMNH DNA Extraction Protocol using Phenol-Chloroform and Isopropanol | <https://github.com/aomlomics/NMNH_DNA_Extraction_Protocol_Phenol-Chloroform/blob/main/NMNH_DNA_Extraction_Protocol_Phenol-Chloroform.md> | 1.0.2 | 2025-12-15 |
| NMNH PCR Protocol 18S rRNA V4 (Stoeck-Piredda) | <https://github.com/aomlomics/NMNH_PCR_Protocol_Eukarya-18S-V4-Stoeck-Piredda/blob/main/NMNH_PCR_Protocol_Eukarya-18S-V4-Stoeck-Piredda.md#nmnh-pcr-protocol-18s-rrna-v4-stoeck-piredda> | 1.0.1 | 2025-12-15 |
| NMNH PCR Protocol COI (Leray-Geller) | <https://github.com/aomlomics/NMNH_PCR_Protocol_Metazoa-COI-Leray-Geller/blob/main/NMNH_PCR_Protocol_Metazoa-COI-Leray-Geller.md> | 1.0.1 | 2025-12-15 |

## RELATED EXTERNAL PROTOCOLS

| PROTOCOL NAME | LINK | ISSUER / AUTHOR | ACCESS DATE |
| ------------ | ------------ | ------------ | ---------- |
| Missing |

### Protocol Revision Record

| VERSION | RELEASE DATE | DESCRIPTION OF REVISIONS |
| ------------- | ------------- | ------------- |
| 1.0.0 | 2025-12-19 | Initial release |

### Acronyms and Abbreviations

| ACRONYM / ABBREVIATION | DEFINITION |
| ------------- | ------------- |
| NMNH | National Museum of Natural History |
| NOAA | National Oceanographic and Atmospheric Administration |
| AOML | Atlantic Oceanographic and Meteorological Laboratory |
| MSU | Mississippi State University |
| NGI | Northern Gulf Institute |
| PCR | Polymerase chain reaction |
| eDNA | environmental DNA |
| NTC | No template control |
| EtOH | Ethanol |

### Glossary

| SPECIALISED TERM | DEFINITION |
| ------------- | ------------- |
| Extraction Blank | A type of negative control to confirm there is no contamination during DNA extractions. Normally an empty is filter extracted and PCR amplified alongside other samples. |
| No Template Control | A type of negative control during PCR to confirm there is no contamination during the PCR process. Normally nuclease-free water is run in place of DNA on a PCR. |

## BACKGROUND

### Summary

This protocol describes steps for performing PCR for [12S rRNA](target_gene) [V5-V6](target_subfragment) marker gene regions of seawater eDNA samples at the National Museum of Natural History (NMNH).

### Method description and rationale

This protocol is used for PCR amplifying the 12S MiFish marker gene regions of environmental DNA.

### Spatial coverage and environment(s) of relevance

This protocol can be used to amplify the 12S marker gene region of any eDNA sample.

## PERSONNEL REQUIRED

One person with molecular biology experience.

### Safety

There are no hazardous chemicals or materials involved in this protocol. Standard lab safety techniques should still be used such as wearing PPE to avoid skin or eye contact.

### Training requirements

Basic molecular biology training is sufficient for this protocol including sterile technique, pipetting small volumes and programming/running [PCR](purpose) thermal cyclers.

### Time needed to execute the procedure

Protocol takes about 4 hours ([240](time_required) minutes) including thermal cycler run time.

## EQUIPMENT

For 96-well Plate:

| DESCRIPTION | PRODUCT NAME AND MODEL | MANUFACTURER | QUANTITY | REMARK |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| **Durable equipment** |
| 100-1000 ul Pipette | Eppendorf Research Plus Adjustable-Volume Pipette | Eppendorf | 1 | Can be substituted with any accurate pipette |
| 10-100 ul Pipette | Eppendorf Research Plus Adjustable-Volume Pipette | Eppendorf | 1 | Can be substituted with any accurate pipette |
| 0.1-2.5 ul Pipette | Eppendorf Research Plus Adjustable-Volume Pipette | Eppendorf | 1 | Can be substituted with any accurate pipette |
| 10-100 ul 8-Channel Pipette | Eppendorf Research Plus 8 Channel Pipette | Eppendorf | 1 | Can be substituted with any accurate pipette |
| 0.5-10 uL 8-Channel Pipette |Eppendorf Research Plus 8 Channel Pipette | Eppendorf | 1 | Can be substituted with any accurate pipette |
| Thermal cycler | Mastercycler Nexus Thermal Cycler | Eppendorf | 1 | Can be substituted with generic |
| **Consumable equipment** |
| Gloves | Nitrile Gloves, Exam Grade, Powder-free | ULINE | 1 | (box) Can be substituted with generic |
| Kim Wipes | KimWipe Delicate Task Wipers | KimTech | 1 | (box) Can be substituted with generic |
| 96-well PCR Plate | | Armadillo PCR Plate, 96-well, clear, clear wells | ThermoFisher | 3 | |
| PCR Plate Seal | AlumaSeal II Sealing Foils for PCR and Cold Storage | VWR | 2 | Can be substituted with generic, can use tightly-fitted strip caps in place of seal |
| 1000µL Filter Tips | OT-2 Filter Tips, 1000µL | Opentrons | 1 | (box) Can be substituted with generic |
| 200µL Filter Tips | OT-2 Filter Tips, 200µL | Opentrons | 2 | (boxes) Can be substituted with generic |
| 10 ul Filter tips | TipOne Pipette Tips, 10 uL | TipOne | 2 | (boxes) Can be substituted with generic |
| (missing) PCR Mix | (missing) | (missing)| 1.2 | (mL) |
| Molecular water | Invitrogen RT-PCR Grade Water | Fisher Scientific | 0.912 | (mL) |
| Forward Primer - 12S MiFish-U-F| 12S MiFish-U-F | IDT | 105 | (ul (10uM)) Primer must be diluted from 100uM stocks to 10uM |
| Reverse Primer - 12S MiFish-U-R | 12S MiFish-U-R | IDT | 105 | (ul (10uM)) Primer must be diluted from 100uM stocks to 10uM |
| **Chemicals** |
| RNase AWAY | RNase AWAY Surface Decontaminant | ThermoFisher Scientific | 1 | (bottle) Used to sterilize lab surfaces and equipment |
| EtOH | Ethanol | Generic Brand | 1 | (wash bottle) Must be molecular grade ethanol |
| DI water | Deionized water | Generic | 900 | (mL) |

- Description: E.g., "filter".
- Product Name and Model: Provide the official name of the product.
- Manufacturer: Provide the name of the manufacturer of the product.
- Quantity: Provide quantities necessary for one application of the standard operating procedure (e.g., number of filters).
- Remark: For example, some of the consumable may need to be sterilized, some commercial solution may need to be diluted or shielded from light during the operating procedure.

## STANDARD OPERATING PROCEDURE

### Protocol

#### Preparation

1. Dilute primers from 100 uM to 10 uM if not already at 10uM.
2. Set up PCR under hood by wiping off all surfaces, pipettes, and racks with RNase AWAY and UV sterilizing for 5-10 mins.
3. Map out order of samples on 96-well PCR plate. Make sure to leave a space for a no template control (NTC) and positive control.

#### PCR

1. Make PCR master mix by adding (missing) ul of the following to each well of PCR plate:

- (missing) ul (missing Master Mix name)
- (missing) ul molecular water
- (missing) ul Fwd primer (10 μM) - 12S MiFish-U-F
- (missing) ul Rev primer (10 μM) - 12S MiFish-U-R

| PCR Primer Name | Direction | Sequence (5’ -> 3’) |
| ----- | ----- | ----- |----|----|
| 12S MiFish-U-F | forward | GTCGGTAAAACTCGTGCCAGC |
| 12S MiFish-U-R | reverse | CATAGTGGGGTATCTAATCCCAGTTTG |

2. Add (missing) ul of sample DNA (or molecular water for NTC) to respective triplicate wells for a total reaction volume of (missing) ul per well. Pipette up and down or vortex to fully distribute DNA into master mix.
3. Seal plate with PCR plate seal or strip caps.
4. Load plate onto thermal cycler and select (missing) program to run the following steps:

| PCR step | Temperature | Duration | Repetition |
| ----- | ----- | ----- | ----- |
| Initial Denaturation | 95°C | 15min | 1x |
| Denaturation | (missing) | (missing) | (missing) |
| Annealing | (missing) | (missing) | (missing) |
| Extension | (missing) | (missing) | (missing) |
| Final Extension | 72°C | 10min | 1x |
| Hold | 4°C | ∞ | |

#### Quality Control

(missing)

### Basic troubleshooting guide

(missing)

## REFERENCES

(missing)

## APPENDIX A: DATASHEETS

(missing)
